src/storage/src/source/source_reader_pipeline.rs

```diff
@@ -1172,7 +1171,13 @@ where
     let mut remap_input = reclock_op.new_input(&remap_trace_updates, Pipeline);
 
     reclock_op.build(move |mut capabilities| {
-        capabilities.clear();
+        // The capability of the output after reclocking the source frontier
```

Comment:
nit: maybe sth like: "A capability for emitting reclocked messages with timestamps of the reclocked time domain."